src/index.js

```diff
@@ -57,6 +57,7 @@ export default ({
   remoteModuleMapUrl,
   useLocalModules,
   appPort,
+  useHost,
 }) => {
```

Comment:
what about adding something like below here. you can probably come up with a better name however
```suggestion
}) => {
  const hostAddress = useHost ? `http://${req.headers.host}` : `http://localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT}`;
```

Reply:
had to put it inside the `get` for the `req` to be accessible but Implemented your changes here: https://github.com/americanexpress/one-app-dev-cdn/pull/14/commits/1c5a17c9779e2264f2584d27afc1031b0fdfd53b